bigbluebutton-html5/imports/ui/components/actions-bar/service.js

```diff
@@ -8,13 +9,31 @@ const getBreakouts = () => Breakouts.find({ parentMeetingId: Auth.meetingID })
   .fetch()
   .sort((a, b) => a.sequence - b.sequence);
 
-const getUsersNotAssigned = (users) => {
-  const breakouts = getBreakouts();
-  const breakoutUsers = breakouts
-    .reduce((acc, value) => [...acc, ...value.users], [])
-    .map(u => u.userId);
-  return users.filter(u => !breakoutUsers.includes(u.intId));
-};
+const getBreakoutUser = user => Users.find({
+  extId: new RegExp(user.userId),
```

Comment:
The regex constructor should be changed to ` Regex('^'+user.userId)` so that it runs quicker.

Reply:
I think the search could also be done with findOne and it would be quicker. You're only ever looking for one record or no record.